pom.xml

```diff
@@ -19,23 +19,78 @@
     <version.org.kie.server>${project.version}</version.org.kie.server>
     <version.org.mockito>3.7.7</version.org.mockito>
     <version.org.projectlombok.lombok>1.18.16</version.org.projectlombok.lombok>
-    <version.org.testcontainers>1.15.3</version.org.testcontainers>
 
+    <!-- Quarkus database setup -->
     <maven.jdbc.db-kind>h2</maven.jdbc.db-kind>
     <maven.jdbc.url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</maven.jdbc.url>
     <maven.jdbc.username>sa</maven.jdbc.username>
     <maven.jdbc.password>sa</maven.jdbc.password>
+    <maven.jdbc.schema>PUBLIC</maven.jdbc.schema>
+
+    <!-- Cargo plugin configuration -->
+    <container.port>8080</container.port>
+    <version.org.wildfly>23.0.2.Final</version.org.wildfly>
+    <cargo.container.id>valid-id-needs-to-be-specified-in-appropriate-container-profile</cargo.container.id>
+    <org.kie.server.datasource.connection.url>jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE</org.kie.server.datasource.connection.url>
+    <org.kie.server.persistence.dialect>org.hibernate.dialect.H2Dialect</org.kie.server.persistence.dialect>
+    <org.kie.server.datasource.driver.class>org.h2.jdbcx.JdbcDataSource</org.kie.server.datasource.driver.class>
+    <org.kie.server.datasource.username>sa</org.kie.server.datasource.username>
+    <org.kie.server.datasource.password>sa</org.kie.server.datasource.password>
+    <kie.server.classifier>ee8</kie.server.classifier>
+
+    <!-- KIE Server configuration -->
+    <maven.kie.server.id>test-kie-server</maven.kie.server.id>
+    <maven.kie.server.context>kie-server</maven.kie.server.context>
+    <maven.kie.server.host>localhost</maven.kie.server.host>
+    <maven.kie.server.http.port>${container.port}</maven.kie.server.http.port>
+    <maven.kie.server.http.url>http://${maven.kie.server.host}:${maven.kie.server.http.port}/${maven.kie.server.context}/services/rest/server</maven.kie.server.http.url>
+    <maven.kie.server.username>kieserver</maven.kie.server.username>
+    <maven.kie.server.password>kieserver1!</maven.kie.server.password>
+
+    <!-- PIM service configuration -->
+    <maven.pim.server.host>localhost</maven.pim.server.host>
+    <maven.pim.server.http.port>8081</maven.pim.server.http.port>
+    <maven.pim.username>admin</maven.pim.username>
+    <maven.pim.password>admin1!</maven.pim.password>
+    <maven.pim.admin.role>admin</maven.pim.admin.role>
+    <maven.pim.server.http.url>http://${maven.pim.server.host}:${maven.pim.server.http.port}</maven.pim.server.http.url>
+    <maven.pim.server.http.health>${maven.pim.server.http.url}/q/health</maven.pim.server.http.health>
 
     <maven.pim.flyway.prefix.version>1.0.0__</maven.pim.flyway.prefix.version>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>jboss-public-repository-group</id>
+      <name>JBoss Public Repository Group</name>
+      <url>https://repository.jboss.org/nexus/content/groups/public/</url>
```

Comment:
That was the idea initially when PIM was migrated to Quarkus, however seems there were many issues with some of the artifacts used by kie-parent and those needed by Quarkus. 
I just added the repository tag to make some artifacts available to the build (can't recall now which ones, but otherwise it will fail to build if those are not installed in your local repo already)

Reply:
Sure, then it's OK.